frontend/nuxt.config.js

```diff
@@ -15,7 +15,7 @@ export default {
       { hid: 'description', name: 'description', content: process.env.npm_package_description || '' }
     ],
     script: [
-      { src: 'https://use.fontawesome.com/releases/v5.0.6/js/all.js' }
+      { src: '/js/all.js' }
```

Comment:
Please roll it back and remove js/all.js. If you want to change this for some reason, submit another issue and PR.